screenshots.js

```diff
@@ -95,14 +109,23 @@ const TMP_CONFIG = { postfix: '.png' };
 			log('-- no diff');
 		}
 	}
+	function onResponse(page, isMaster) {
+		const pageUrls = Object.keys(config).map(path => `http://${PR}.talend.surge.sh/${path}`);
+		page.on('response', async res => {
+			if (pageUrls.includes(res.url()) && res.status() === 404) {
+				console.error(`Surge is not ready yet ${isMaster ? 'for Master' : `for the PR ${PR}`}: ${res.url()}`);
```

Comment:
One of the condition to execute this is that the requested url is one of the PR's one, so it conflicts with the ternary checking `isMaster` here.
With my previous comment, I still don't understand what it is supposed to do exactly. It should be reworked or explained.